tools/sidekick/config.js

```diff
@@ -15,4 +15,5 @@ window.hlx.initSidekick({
   project: 'Adobe I/O',
   host: 'www.adobe.io',
```

Comment:
should this still be www.adobe.io?  We moved the name to developer.adobe.com.  In theory.  Both names are bound to same service endpoint in Fastly so there is no real difference there.  Will it make any difference on your side?